appeals/web/src/server/views/app/components/file-uploader.component.njk

```diff
@@ -5,6 +5,7 @@
 
 	<div class="govuk-grid-row pins-file-upload" data-next-page-url="{{ params.nextPageUrl }}"
 		 data-case-id="{{ params.caseId }}" data-folder-id="{{ params.folderId }}"
+		 data-document-type="{{ params.documentType }}" data-document-stage="{{ params.documentStage }}"
 		 data-blob-storage-host="{{ params.blobStorageHost }}" data-blob-storage-container="{{ params.blobStorageContainer}}"
```

Comment:
Just wondering where all the metadata in these attributes is used. If it's used for any sort of validation then it should probably be stored in config as html data like this can be easily manipulated by the user

Reply:
These are derived from the folder documents are uploaded to, and saved in the documentVersion table. The folders are created from config, but agree may need some extra checks to validate on the API side... I'll look into it.

Reply:
I think I'll leave it when we implement the UI side of things...